apps/metadata-editor/src/app/dashboard/search-filters-summary/search-filters-summary.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, Input } from '@angular/core'
+import { CommonModule } from '@angular/common'
+import { SearchFacade, SearchService } from '@geonetwork-ui/feature/search'
+import { map } from 'rxjs'
+import { SearchFiltersSummaryItemComponent } from '../search-filters-summary-item/search-filters-summary-item.component'
+import { TranslateModule } from '@ngx-translate/core'
+
+@Component({
+  selector: 'md-editor-search-filters-summary',
+  imports: [CommonModule, SearchFiltersSummaryItemComponent, TranslateModule],
+  templateUrl: './search-filters-summary.component.html',
+  styleUrls: ['./search-filters-summary.component.css'],
+  standalone: true,
+})
+export class SearchFiltersSummaryComponent {
+  @Input() searchFields: string[] = []
+
+  searchFilterActive$ = this.searchFacade.searchFilters$.pipe(
+    map((filters) => this.hasNonEmptyValues(filters))
+  )
```

Comment:
I'm wondering if instead of taking a list of filters as input we can simply listen for the search filters and show everyone of them. This means that if another filter (which does not appear in the advanced filters above) is active then it will also show up. What do you think?

Reply:
I gave it a try, but had some difficulties with the fact that the field names do not always correspond to the ES field names in the filters. I'm also wondering if it is not more intuitive from a user perspective to only display the summary badges for the filters visible in the filter component above. For example, on the `myRecords` page the filter `{ owner: user.id }` should always be present.